book/IMaEA.md

```diff
@@ -6592,7 +6592,7 @@ I abandoned Jenkins for several reasons:
 3. I didn't understand how to trigger Jenkins from GitHub.
 4. I didn't like the Cygwin ssh server which runs the bash shell.
 
-I decided to forget about Jenkins and focus on the ssh script.  So build.sh, parses its command arguments, writes the build script and transfers it by ssh to the appropriate VM.  On the machine, rmillsmm, I have VM such as rmillsmm-w10, rmillsmm-ubuntu, rmillsmm-solaris and so on.  On Windows, I use the excellant bitvise ssh server and the native server on other platforms.
+I decided to forget about Jenkins and focus on the ssh script.  So build.sh, parses its command arguments, writes the build script and transfers it by ssh to the appropriate VM.  On the machine, rmillsmm, I have VM such as rmillsmm-w10, rmillsmm-ubuntu, rmillsmm-solaris and so on.  On Windows, I use the excellent bitvise ssh server and the native server on other platforms.
```

Comment:
This should probably be `Bitvise` and possibly w/ a link.

If I can declare that out of scope for the PR and leave it for someone to do as a follow-up, I'd like to.